website/docs/user_guide/personal-access-token.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: How to create Personal Access Tokens
+---
+
```

Comment:
Is this a duplicate of how-to-create-personal-access-tokens?

Reply:
Yeah, removed